### ISSUE TITLE
Add Ethereum support for Bountysource

### ISSUE THREAD

**Issue by yong638** [bountysource/core#1143]
It's about time we accept ether/ETH as a payment method.

**Comment by Eothred** [comment]
As they commented in #714 and #715 they were not planning on supporting other cryptos per Nov 2015 (which I hope has changed by now).

**Comment by kaiansaari** [comment]
You're right @Eothred, support for ETH & ERC20 is coming to Bountysource.
